### PR TITLE
Update adafruit libraries to latest release

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ env:
     - PLATFORMIO_CI_SRC=./
 
 install:
-    - pip install -U platformio
+    - pip install -U platformio<4.0.0
```
Comment (bertrik) [inline .travis.yml]: What's wrong with platformio 4.0.0 and higher?

Comment (dirkmueller) [reply .travis.yml]: See.commit.message. the new version rejects the lang env which causes build to fail in travis.

I forgot to add quotes so it now also fails. Will fix when I am back at the computer

Comment (bertrik) [reply .travis.yml]: Ah ok, I've been seeing warnings about the 'lang' variable for a long time. If this is now an error in platformio 4.0.0, might be best to stick to the older version for now, so we can fix the underlying problem in the meantime.

Comment (dirkmueller) [reply .travis.yml]: As you can see this seems to fix the Travis failure.

Comment (dirkmueller) [reply .travis.yml]: s,fix,workaround,

**Comment by dirkmueller** [comment]
@here any chance that we can merge this?

**Comment by ricki-z** [comment]
Sorry, we had some holiday.
And there were some major changes to the main script that needed to be pushed before.

**Pull request merged**